build_tools/cmake/iree_copts.cmake

```diff
@@ -346,3 +346,28 @@ if(IREE_ENABLE_EMITC)
   )
   add_definitions(-DIREE_HAVE_EMITC_DIALECT)
 endif()
+
+#-------------------------------------------------------------------------------
+# Third party: SPIRV-Cross
+#-------------------------------------------------------------------------------
+
+if(${IREE_TARGET_BACKEND_METAL-SPIRV})
+  set(SPIRV_CROSS_ENABLE_MSL ON CACHE BOOL "" FORCE)
+  set(SPIRV_CROSS_ENABLE_GLSL ON CACHE BOOL "" FORCE) # Required to enable MSL
+
+  set(SPIRV_CROSS_EXCEPTIONS_TO_ASSERTIONS OFF CACHE BOOL "" FORCE)
+  set(SPIRV_CROSS_CLI OFF CACHE BOOL "" FORCE)
+  set(SPIRV_CROSS_ENABLE_TESTS OFF CACHE BOOL "" FORCE)
+  set(SPIRV_CROSS_SKIP_INSTALL ON CACHE BOOL "" FORCE)
+
+  set(SPIRV_CROSS_ENABLE_HLSL OFF CACHE BOOL "" FORCE)
+  set(SPIRV_CROSS_ENABLE_CPP OFF CACHE BOOL "" FORCE)
+  set(SPIRV_CROSS_ENABLE_REFLECT OFF CACHE BOOL "" FORCE)
+  set(SPIRV_CROSS_ENABLE_C_API OFF CACHE BOOL "" FORCE)
+  set(SPIRV_CROSS_ENABLE_C_API OFF CACHE BOOL "" FORCE)
+  set(SPIRV_CROSS_ENABLE_UTIL OFF CACHE BOOL "" FORCE)
+
+  list(APPEND IREE_COMMON_INCLUDE_DIRS
+    ${PROJECT_SOURCE_DIR}/third_party
```

Comment:
is this required? I really would like to remove IREE_COMMON_INCLUDE_DIRS entirely - you should add this to the cc_library that actually pulls in these files or use the repository-relative path (third_party/spirv-cross/....) in the include

Reply:
Moved to the library that uses this.